Data/Resources/README.md

```diff
@@ -0,0 +1,16 @@
+# Resources README
+
+Folders in "Resources" are as follows:
+- 2021_release: contains resource files used in 2021_release, if they have been updated. Not updated files remain in main folder.
```

Comment:
Same comment: 

2021_release: contain older versions of resource files used in prior 2021 MWI release. Files without data updates remain in main folder.